sdk/CMakeLists.txt

```diff
@@ -149,13 +149,13 @@ endif()
 if (USE_ITOF)
 	
 	if ( UNIX )
-    		add_library(libtofi_compute SHARED IMPORTED)
```

Comment:
Why static and not shared? I know you mention this some time ago but I forgot (sorry).